webapp/channels/src/components/copy_button.tsx

```diff
@@ -1,28 +1,53 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import type {Placement} from '@floating-ui/react';
 import classNames from 'classnames';
 import React, {useRef, useState} from 'react';
-import {Tooltip} from 'react-bootstrap';
-import {FormattedMessage} from 'react-intl';
+import {useIntl} from 'react-intl';
 
-import OverlayTrigger from 'components/overlay_trigger';
+import useTooltip from 'components/common/hooks/useTooltip';
```

Comment:
i know i mentioned to use `useTooltip` earlier, but after our recent conversation we are moving all tooltips towards `WithTooltip` its API is much simpler as well, are you open to making a small change?